sympy/geometry/point.py

```diff
@@ -233,21 +413,35 @@ def ambient_dimension(self):
 
     @property
     def orthogonal_direction(self):
-        """Returns a non-zero point in a direction
-        orthogonal to `self`."""
+        """Returns a non-zero point in the x-y plane that is
```

Comment:
I guess I'm confused as to why you changed it from what it was.  It was a fully-general function.